### PR TITLE
feat: frontend gradient option

### PR DESCRIPTION
**Pull request by arcbtc**
Adds a gradient option to themes
Bit bloaty js, but performance not impacted

### CODE REVIEW

**Comment by motorina0** [comment]
Love it!
Why still in draft?

File: lnbits/static/js/base.js [hunk 1]
```diff
@@ -422,6 +422,18 @@ window.LNbits = {
       converter.setFlavor('github')
       converter.setOption('simpleLineBreaks', true)
       return converter.makeHtml(text)
+    },
+    hexToRgb: function (hex) {
```
Comment (dni) [inline lnbits/static/js/base.js:426]: i dont think wrapping this into `LNbits.utils` makes sense, just use it directly

**Comment by arcbtc** [comment]
> Love it! Why still in draft?

Because its pure spaghetti!

File: lnbits/static/js/base.js (hunk 1 above)
Comment (arcbtc) [reply lnbits/static/js/base.js:426]: Thought this way it would be useful for others

**Comment by motorina0** [comment]
- I haven't managed to get the gradient effect:
image

 - Mobile looks strange when the left-side menu is visible:
image

- it is not clear in the UI when the gradient toggle is on or off (button state does not change)

**Review by dni — APPROVED** [review]

**Comment by prusnak** [comment]
This PR should not modify package-lock.json because it does not touch package.json

Please revert that change

**Comment by dni** [comment]
> This PR should not modify package-lock.json because it does not touch package.json
> 
> Please revert that change

done

**Pull request merged**